### PR TITLE
Simple cov

### PR DESCRIPTION
**Pull request by ryandbarnett**
Setup simple cov

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+coverage/
```
Comment (dhalverson) [inline .gitignore:1]: Great adding this to ignore

**Review by dhalverson — APPROVED** [review]
Looks good

File: .gitignore (hunk 1 above)
Comment (ryandbarnett) [reply .gitignore:1]: Thanks

**Review by hashmaster3k — APPROVED** [review]
Thanks for going through and adding all of this.

**Pull request merged**